bluebottle/initiatives/admin.py

```diff
@@ -212,8 +212,6 @@ def valid(self, obj):
             format_html_join("", "<li>{}</li>", ((value,) for value in errors))
         )
 
-    autocomplete_fields = ['activity_managers']
```

Comment:
this will then show a long list with all users where you can select multiple, right? If you have a lot of users this will make these pages really slow.